cmd/multiclusterapp.go

```diff
@@ -138,17 +162,59 @@ func MultiClusterAppCommand() cli.Command {
 						Name:  "set",
 						Usage: "Set answers for the template, can be used multiple times. Example: --set foo=bar",
 					},
+					cli.StringSliceFlag{
+						Name: "role,r",
+						Usage: "Set roles required to view the app. Specified roles on upgrade will override all " +
+							"the original roles. Leave it empty to keep current roles",
+					},
 					cli.BoolFlag{
 						Name:  "show-versions,v",
 						Usage: "Display versions available to upgrade to",
 					},
+				},
+			},
+			cli.Command{
+				Name:        "add-project",
+				Usage:       "Add target projects to a multi-cluster app",
+				Action:      addMcappTargetProject,
+				Description: "Examples:\n #Add 'p1' project in cluster 'mycluster' to target projects of a multi-cluster app named 'myapp'\n rancher multiclusterapp add-project myapp mycluster:p1\n",
+				ArgsUsage:   "[APP_NAME/APP_ID, CLUSTER_NAME:PROJECT_NAME/PROJECT_ID...]",
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "answers,a",
+						Usage: "Path to an answers file that provides overrided answers for the new target projects, the format of the file is a map with key:value. Supports JSON and YAML",
+					},
+					cli.StringFlag{
+						Name:  "values",
+						Usage: "Path to a helm values file that provides overrided answers for the new target projects",
+					},
 					cli.StringSliceFlag{
-						Name: "target,t",
-						Usage: "Target project names/ids to upgrade. Specified targets on upgrade will override all " +
-							"the original targets. Leave it empty to keep current targets",
+						Name:  "set",
+						Usage: "Set overrided answers for the new target projects",
 					},
 				},
 			},
+			cli.Command{
+				Name:        "delete-project",
+				Usage:       "Delete target projects from a multi-cluster app",
+				Action:      deleteMcappTargetProject,
+				Description: "Examples:\n #Delete 'p1' project in cluster 'mycluster' from target projects of a multi-cluster app named 'myapp'\n rancher multiclusterapp delete-project myapp mycluster:p1\n",
+				ArgsUsage:   "[APP_NAME/APP_ID, CLUSTER_NAME:PROJECT_NAME/PROJECT_ID...]",
+			},
+			cli.Command{
+				Name:        "add-member",
```

Comment:
Should add-member also have the member-access-type parameter?

Reply:
Access-type is included in the command args. The usage of add-member is:
```
"[APP_NAME/APP_ID, USER_NAME/USER_ID, ACCESS_TYPE]"
rancher mcapp add-member user1 read-only
```
Or do you think it's better if it goes as
```
rancher mcapp add-member --member-access-type read-only u1 u2 u3...
```

Reply:
If so, I would prefer using rancher mcapp add-member myapp owner user1 user2 ... Put the scaleable parameters in the end then you can add multiple user in this command.

Reply:
Updated

Reply:
Agreed on keeping it this way, it's how project and cluster members work.